### PR TITLE
Removes Romerol from the Final Objectives

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Removes Romerol from the Final Objectives

## Why It's Good For The Game

oranges has spoken

## Changelog

:cl:
del: Removes Romerol from the Final Objectives
/:cl:

### CODE REVIEW

**Comment by SuperNovaa41** [comment]
:( romerol is probably like my all time favourite way to have a chaotic round, its generally far more enjoyable than like: war ops, wizards, murderbone, and pretty much anything that will disrupt a round. It gives people a chance to play again + like this is super late round anyways, why mess with it at all?

**Comment by Shadowflame909** [comment]
I've had unimpeded traitor rounds and have not reached that final objective.

What is the reason for this?

**Comment by bluetshirtguy** [comment]
The first one is kinda strange since I've heard a lot of talks on adjusted the amount of TC gained over time or started with but making the starfury is good, it's insane traitors have that option to overshadow themselves in the first place.

**Comment by optimumtact** [comment]
> I've had unimpeded traitor rounds and have not reached that final objective.
> 
> What is the reason for this?

I do not like romerol and I do not like zombies

**Comment by realthomas69** [comment]
> I do not like romerol and I do not like zombies

could you do the bare minimum of explaining why? this does nothing but make people dislike you guys more

**Comment by Shadowflame909** [comment]
Replace romerol with a rogue agent smith clone virus that has gained access into the real (real??) world

Now thats sci-fi.

**Comment by Donpedrito** [comment]
I’m not a fan of putting a hard timer on that kind of thing, it feels like it would defeat the purpose of progression traitors.
Also at the same time, as a Bagil player (biased) I worry that it’d actually make the Starfury more common on servers with longer rounds if the restriction was time instead of points.
If possible please consider keeping the solution of doubling the points needed as permanent, as I believe it achieves the same purpose of keeping the Starfury from appearing until about 2 hours in, but also gives traitors more agency to somewhat affect the time by doing objectives well.

**Comment by optimumtact** [comment]
> > I do not like romerol and I do not like zombies
> 
> could you do the bare minimum of explaining why? this does nothing but make people dislike you guys more

oh no, what a shame

File: code/modules/antagonists/traitor/objectives/final_objective/final_objective.dm
Comment (Time-Green) [inline code/modules/antagonists/traitor/objectives/final_objective/final_objective.dm:10]: Almost 5 hours for a final objective to become available? I’m not sure if this’ll ever happen (except for the one guy playing campbell at 3AM)

**Comment by Dorsisdwarf** [comment]
Romerol SUCKS and should be in the game as little as possible

**Review by Watermelon914 — CHANGES_REQUESTED** [review]
please present data of players getting starfury at the 1 hr mark or I'm going to ask you to remove the changes you've made to the final objective minimum progression time count

**Comment by ANAME593** [comment]
>people were calling starfury at like, an hour in

there is literally no way at all this has happened even if you speedrun it takes up to around 90m or so

**Comment by Mothblocks** [comment]
@Watermelon914 tell them about what the data showed dear

**Comment by SomeDongus** [comment]
> > I've had unimpeded traitor rounds and have not reached that final objective.
> > What is the reason for this?
> 
> I do not like romerol and I do not like zombies

kys

**Comment by Iamgoofball** [comment]
@Watermelon914 You're outruled here, this is an oranges request.

**Comment by Iamgoofball** [comment]
According to the data, someone managed an 83 minute romerol.
ApplicationFrameHost_XkQ2U8RwVl

Your code doesn't support actual hard minimum time requirements, regretfully, otherwise I would have made 120 minutes the hard minimum.

**Comment by Iamgoofball** [comment]
Discord_awyoDUEY3Y
Direct statement from the orange man regarding how early it should unlock. If you can make code to make it do this, then I'll remove my changes. Otherwise, tell me what minute requirement to put in that actually equates to 120 minutes as per the OP.

**Comment by Iamgoofball** [comment]
I've adjusted it to 160 for now to cover the earliest recorded case, but if it's possible to get 120 progression sooner then we may need to adjust it up further.

**Comment by Watermelon914** [comment]
> I've adjusted it to 160 for now to cover the earliest recorded case, but if it's possible to get 120 progression sooner then we may need to adjust it up further. Hence why I asked you in the OP of this PR that you clearly did not read to tell me what number should go here.

no, this is still the completely wrong way to fix this. Remove the changes completely and let me handle balancing if we get another case under 90 minutes. After analysing the data, that 81 minute is an outlier because there was no security to stop them from rapidly completing all their objectives.

**Comment by Shadowflame909** [comment]
If you balance off of the best case scenario, then you wont ever see the starfury in the regular average gameplay loop.

File: code/modules/antagonists/traitor/objectives/final_objective/final_objective.dm
Comment (Mothblocks) [inline code/modules/antagonists/traitor/objectives/final_objective/final_objective.dm:4]:
```suggestion
		/datum/traitor_objective/final/battlecruiser = 1,
```

Trailing commas good

**Comment by SuperNovaa41** [comment]
Ok awesome, now actually explain why this is good for the game

**Comment by Acer202** [comment]
When I played a month and a half ago, rounds on Sybil averaged something like 50 minutes to barely past 1 hour. I didn't see a single zombie outbreak that entire time, and I wish I did! Did round lengths change recently, or did progression traitor really slow things down?

**Comment by ANAME593** [comment]
since when do final objectives spawn there has only been 1 or 2 cases of them spawning and i've hit Max rep past the 1 hour mark and i haven't gotten it

**Comment by moocowswag** [comment]
I would support this if it meant zombies as a theme were preserved and updated to have much more appealing gameplay as their own antag.

**Comment by Mothblocks** [comment]
You guys all know necrotic revival is still a thing right

**Comment by Shadowflame909** [comment]
> You guys all know necrotic revival is still a thing right

With all that it requires and the fact that one of the head maintainers doesn't like zombies. It won't be a thing for long.

**Comment by TheVekter** [comment]
oranges says on the forums that the issue is that zombies don't have a way to end the round.

image

**Comment by ANAME593** [comment]
> oranges says on the forums that the issue is that zombies don't have a way to end the round.
 if there  are almost no crew left a nuke gets sent by centcom kind of like nar'sie

**Comment by Iamgoofball** [comment]
just waiting on a merge

**Comment by Shadowflame909** [comment]
> just waiting on a merge

That the playerbase hopes never comes

**Comment by Hamurlik** [comment]
Fuck oranges

**Comment by Mothblocks** [comment]
Aren't you that guy I've banned like 30 times

**Comment by timothymtorres** [comment]
This is a bad change.  Romerol was already super rare even before the traitor reputation rework.  This is just making it impossible to reach during a normal round considering how fast the shuttle gets called because people are bored.  We shouldn't be balancing things based on a speedrunner who rushes to complete objectives on lowpop with no security present.  That is not indicative of a normal round.

**Comment by Watermelon914** [comment]
> This is a bad change. Romerol was already super rare even before the traitor reputation rework. This is just making it impossible to reach during a normal round considering how fast the shuttle gets called because people are bored. We shouldn't be balancing things based on a speedrunner who rushes to complete objectives on lowpop with no security present. That is not indicative of a normal round.

yes it's a weak solution, but romerol needs some endstate to be worth keeping in the game (which it currently does not have)

**Comment by optimumtact** [comment]
you need to  be able to reverse the zombie virus with engagement through virology once you get a sample of enough unique zombies.

it also needs the zombie win state to be improved so that the round is ended if zombies take over a major portion of the crew.

**Comment by Mothblocks** [comment]
I wanted this for aliens, I think it can potentially work here as well, feel free to offer thoughts:

1. After a *significant* (like, 70-80%) portion of the crew is dead/zombied, spawn an ERT. The point of the ERT is primarily to call the shuttle, but they still have weaponry and whatnot of course.
2. After a good bit of time after *that*, if they didn't call the shuttle (lame!), auto-call shuttle with red alert.

**Comment by Toastgoats** [comment]
Imho just have the shuttle autocalled if a certain percentage of living players are zombies. Sort of like how space dragon force calls the shuttle once it wins

**Comment by Iamgoofball** [comment]
can we get this merged until someone reworks romerol

**Comment by squi1-rasb** [comment]
If this PR reaches a month of being open goof will evolve

**Comment by optimumtact** [comment]
I have changed my mind, but, I do want to see mechanics for countering zombies in the near future, so the round doesn't stalemate with zombies

**Pull request closed without merging**
